### PR TITLE
Changes to correct for different formats of the returned certificate.

### PR DESCRIPTION
**Pull request by markroberts0830**
Had to make changes as a result of what was found at a UK customer. Different versions of Venafi will either return 'CertificateData=' or 'CertificateData:'.

### CODE REVIEW

**Comment by mbeshara** [comment]
@UrbanCode/plugins

File: src/main/zip/info.xml [hunk 1]
```diff
@@ -24,6 +28,12 @@
   </author>
 
   <!--
+<<<<<<< HEAD
+  **intergration type IS Required**
+  The integration type section identifies where the plugin fits into the process
+  excepted types are SCM, Build, Artifact, Automation, Deploy
+  For example:
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: Unresolved merge conflicts. These are causing the plugin build to fail.

Please see the following Git documentation to help explain how to handle merge conflicts:
https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line

File: src/main/zip/info.xml [hunk 2]
```diff
@@ -44,7 +55,10 @@
 
   <!--
   **source is NOT required**
+<<<<<<< HEAD
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: More unresolved merge conflicts.

File: src/main/zip/info.xml [hunk 3]
```diff
@@ -54,7 +68,10 @@
   -->
   <!--
   **license type is NOT required**
+<<<<<<< HEAD
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: More unresolved merge conflicts.

File: src/main/zip/info.xml [hunk 4]
```diff
@@ -65,10 +82,15 @@
   -->
   <!--
   **tool-description IS required**
+<<<<<<< HEAD
+  The tool-description section is used to summarize the software
+  the plugin was created to integrate with.
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: Unresolved merge conflicts.

File: src/main/zip/info.xml [hunk 5]
```diff
@@ -80,13 +102,20 @@
 
   <!--
   **related-info is NOT required**
+<<<<<<< HEAD
+  The releated-info section is used to define links which may be useful to users of the plugin
+  but don't fall into the release-notes or tool-description section.
+  For example: releated-info can be links to pdf documentation, help videos related to plugin setup or the product's
+  website.
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: Unresolved merge conflicts.

File: src/main/zip/info.xml [hunk 6]
```diff
@@ -96,12 +125,18 @@
...
+<<<<<<< HEAD
+=======
 
+>>>>>>> bcd33dde3bb541b4afbdb2de8f831a463496f986
   The meta-html section is used define the meta description and the meta keywords of the plugin page.The meta
   description tag allows you to influence the description of your page in the web crawlers that support the
   tag The meta keywords tag allows   you to provide additional text for crawler-based search engines to index
   along with your body copy.
+<<<<<<< HEAD
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: Unresolved merge conflicts.

File: src/main/zip/info.xml [hunk 7]
```diff
@@ -117,6 +152,12 @@
   <release-notes>
     <!--
       **release-note IS required**
+<<<<<<< HEAD
+      The plugin-version name must match the plugin version found in the plugin.xml file
+      The release-note should document the changes, improvements, or bug fixes introduced
+      with the new plugin version
+      The release-note may contain text or reference a file in the plugin zip package
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: Unresolved merge conflicts.

File: src/main/zip/info.xml [hunk 8]
```diff
@@ -125,16 +166,23 @@
...
+>>>>>>> bcd33dde3bb541b4afbdb2de8f831a463496f986
       For example your release note could be:
       Support for was added for version 10.0 for software X
       or
       releaseNotes/release1.html (where release1.html contains html formatted version information)
+<<<<<<< HEAD
+      To add a new release note add this with a description in it of the changes:
+      <release-note plugin-version="1">....</release-note>
+      Our automated release process will replace 'LATEST' with the released version.
+=======
```
Comment (jrbarto) [inline src/main/zip/info.xml]: Unresolved merge conflicts.

File: src/main/zip/upgrade.xml [hunk 9]
```diff
@@ -16,6 +28,9 @@
     <migrate to-version="19">
         <migrate-command name="update default values"/>
     </migrate>
+<<<<<<< HEAD
+=======
```
Comment (jrbarto) [inline src/main/zip/upgrade.xml]: Unresolved merge conflicts.

**Comment by leosraj** [comment]
Hi Jeff,

I see the build resolved in the branch available in :
https://github.com/kidharb/Venafi-UCD/tree/correction




Thanks.



LEO J. SELVARAJ
IBM Certified IT Specialist – Level Two
IBM Systems Middleware Services


                                                                                     
                                                                                     
                                                                                     
                                                                                     
                                                                                     
                                                                                     
 2D barcode -  Phone: 1-720-395-1729 | Mobile: 1-248-837-9233                        
 encoded with  E-mail: leosraj@us.ibm.com                                            
 contact       ISSW:                                                                 
 information   www.ibm.com/developerworks/websphere/services/                        
                                                                                     
                                                                                     
                                                                                     








From:	Jeffrey Barto <notifications@github.com>
To:	UrbanCode/Venafi-UCD <Venafi-UCD@noreply.github.com>
Cc:	Subscribed <subscribed@noreply.github.com>
Date:	10/02/2019 04:39 PM
Subject:	[EXTERNAL] Re: [UrbanCode/Venafi-UCD] Changes to correct for
            different formats of the returned certificate. (#2)



@jrbarto requested changes on this pull request.


It seems that the entire change is unresolved merge conflicts. There
doesn't seem to be anything functionally affected by this change. However,
the plugin will not be able to build if we merge this pull request in.





In src/main/zip/info.xml:


> @@ -24,6 +28,12 @@
   </author>

   <!--
+<<<<<<< HEAD
+  **intergration type IS Required**
+  The integration type section identifies where the plugin fits into the
process
+  excepted types are SCM, Build, Artifact, Automation, Deploy
+  For example:
+=======



Unresolved merge conflicts. These are causing the plugin build to fail.


Please see the following Git documentation to help explain how to handle
merge conflicts:
https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line





In src/main/zip/info.xml:


> @@ -44,7 +55,10 @@

   <!--
   **source is NOT required**
+<<<<<<< HEAD
+=======



More unresolved merge conflicts.





In src/main/zip/info.xml:


> @@ -54,7 +68,10 @@
   -->
   <!--
   **license type is NOT required**
+<<<<<<< HEAD
+=======



More unresolved merge conflicts.





In src/main/zip/info.xml:


> @@ -65,10 +82,15 @@
   -->
   <!--
   **tool-description IS required**
+<<<<<<< HEAD
+  The tool-description section is used to summarize the software
+  the plugin was created to integrate with.
+=======



Unresolved merge conflicts.





In src/main/zip/info.xml:


> @@ -80,13 +102,20 @@

   <!--
   **related-info is NOT required**
+<<<<<<< HEAD
+  The releated-info section is used to define links which may be useful to
users of the plugin
+  but don't fall into the release-notes or tool-description section.
+  For example: releated-info can be links to pdf documentation, help
videos related to plugin setup or the product's
+  website.
+=======



Unresolved merge conflicts.





In src/main/zip/info.xml:


>    The meta-html section is used define the meta description and the meta
keywords of the plugin page.The meta
   description tag allows you to influence the description of your page in
the web crawlers that support the
   tag The meta keywords tag allows   you to provide additional text for
crawler-based search engines to index
   along with your body copy.
+<<<<<<< HEAD
+=======



Unresolved merge conflicts.





In src/main/zip/info.xml:


> @@ -117,6 +152,12 @@
   <release-notes>
     <!--
       **release-note IS required**
+<<<<<<< HEAD
+      The plugin-version name must match the plugin version found in the
plugin.xml file
+      The release-note should document the changes, improvements, or bug
fixes introduced
+      with the new plugin version
+      The release-note may contain text or reference a file in the plugin
zip package
+=======



Unresolved merge conflicts.





In src/main/zip/info.xml:


>        For example your release note could be:
       Support for was added for version 10.0 for software X
       or
       releaseNotes/release1.html (where release1.html contains html
formatted version information)
+<<<<<<< HEAD
+      To add a new release note add this with a description in it of the
changes:
+      <release-note plugin-version="1">....</release-note>
+      Our automated release process will replace 'LATEST' with the
released version.
+=======



Unresolved merge conflicts.





In src/main/zip/upgrade.xml:


> @@ -16,6 +28,9 @@
     <migrate to-version="19">
         <migrate-command name="update default values"/>
     </migrate>
+<<<<<<< HEAD
+=======



Unresolved merge conflicts.


—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.

**Pull request closed without merging**